test/lib/flat_mutation_reader_assertions.hh

```diff
@@ -543,8 +544,30 @@ class flat_reader_assertions_v2 {
     flat_mutation_reader_v2 _reader;
     dht::partition_range _pr;
     bool _ignore_deletion_time = false;
+    tombstone _rt{};
```

Comment:
This is a weird syntax for member default initialization...
I'd expect it to be written as `tombstone _rt = {};`,
but even that is not needed.  Since struct tombstone has a default constructor and it is not trivial,
``tombstone _rt;` would do.